sopel/coretasks.py

```diff
@@ -90,11 +137,15 @@ def startup(bot, trigger):
 
     if bot.config.core.throttle_join:
         throttle_rate = int(bot.config.core.throttle_join)
+        throttle_wait = max(bot.config.core.throttle_wait, 1)
         channels_joined = 0
         for channel in bot.config.core.channels:
             channels_joined += 1
             if not channels_joined % throttle_rate:
-                time.sleep(1)
+                LOGGER.info(
```

Comment:
```suggestion
                LOGGER.debug(
```

Reply:
In my usage of this feature, I find it quite confusing not to have this information: if you have a lot of channels to join, it feels like the bot is blocked doing nothing. This INFO log can help with that, making sure the user knows there is something going on, and that the wait time is actually expected!

What are your thought on that point?

Reply:
My thoughts are: Output one INFO log line if throttling is enabled saying so, before JOIN processing starts, and make this one a DEBUG.

If you want to output more INFO-level stuff at startup, output to the log when the bot joins a channel. 😝 It's true that right now it just says "Connected." and shuts up. Maybe take inspiration [from Bucket](https://github.com/zigdon/xkcd-Bucket/blob/50a9f13b4c875aa7385144ba27df04877c7aaef0/bucket.pl#L2529-L2532)? (Yes, I know it's "ew, Perl", but it's sensible log output.)

Reply:
> My thoughts are: Output one INFO log line if throttling is enabled saying so, before JOIN processing starts, and make this one a DEBUG.

That's clever and better. I'll do that.